Kernel/Devices/Audio/AC97.cpp

```diff
@@ -274,7 +274,9 @@ ErrorOr<void> AC97::write_single_buffer(UserOrKernelBuffer const& data, size_t o
 void AC97::AC97Channel::handle_dma_stopped()
 {
     dbgln_if(AC97_DEBUG, "AC97 @ {}: channel {}: DMA engine has stopped", m_device.pci_address(), name());
-    VERIFY(m_dma_running);
+    // QEMU might send spurious interrupts while we're not running, so we don't want to panic here.
```

Comment:
Small nitpick: We usually start a note with `// Note: ...`, to be able to easily track them :)

Reply:
I think we should put that in the code style at this point, I've seen people both do and not do it